dashboards/google-compute-engine/metadata.yaml

```
@@ -23,3 +23,11 @@ sample_dashboards:
     related_integrations:
       - id: gce
         platform: GCP
+  -
+    category: Google Compute Engine
+    id: gce-reservations-monitoring-dashboard
```

```suggestion
    id: reservations-monitoring
```
This should match the filename of the json spec